lab_4_retrieval_w_clustering/main.py

```diff
@@ -140,6 +199,13 @@ def get_vectors(self, indices: list[int] | None = None) -> list[tuple[int, Vecto
         Returns:
             list[tuple[int, Vector]]: List of index and vector for documents.
         """
+        if not indices:
+            return list(self.__vectors.items())
+        unique = []
```

Comment:
set?

Reply:
no cause i can't change the order of elements inside indices